buildout_1.rst

```diff
@@ -16,9 +16,9 @@ Topics covered:
 
 .. only:: not presentation
 
-    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ composes your application for you, according to your rules.
+    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ configures your application for you, according to your rules.
 
-    To compose your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
+    To configure your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
     Buildout downloads the eggs you requested and resolves all dependencies. You might need five different eggs, but in the end, Buildout has to install 300 eggs, all with the correct version in order to resolve all the dependencies.
```

Comment:
I am not alone: http://reinout.vanrees.org/weblog/2015/06/16/djangorecipe-gunicorn.html
buildout itself does use the term assembly/assembles https://pypi.python.org/pypi/zc.buildout/2.4.4

If you tell one application where to find stuff then it is clearly configuration, but you are usually telling multiple parts where they find each other, to me this is a composition of multiple parts to work well together.
This is also the way I have always presented buildout in philips and my trainings.